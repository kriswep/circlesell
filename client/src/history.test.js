/* globals test expect */
import history from './history';

test('history should be created', () => expect(history).toMatchSnapshot());
