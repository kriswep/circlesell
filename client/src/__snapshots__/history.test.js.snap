// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`history should be created 1`] = `
Object {
  "action": "POP",
  "block": [Function],
  "createHref": [Function],
  "go": [Function],
  "goBack": [Function],
  "goForward": [Function],
  "length": 1,
  "listen": [Function],
  "location": Object {
    "hash": "",
    "key": undefined,
    "pathname": "/",
    "search": "",
    "state": undefined,
  },
  "push": [Function],
  "replace": [Function],
}
`;
